x-pack/plugins/security_solution/cypress/e2e/timelines/export.cy.ts

```diff
@@ -37,4 +43,23 @@ describe('Export timelines', () => {
       cy.wrap(response?.body).should('eql', expectedExportedTimeline(this.timelineResponse));
     });
   });
+
+  it('Bulk export all created timeline', function () {
+    visitWithoutDateRange(TIMELINES_URL);
+    cy.get(TIMELINE_CHECKBOX(this.timelineId)).should('exist'); // wait for all timelines to be loaded
+    selectAllTimelines();
+    exportSelectedTimelines();
+
+    const expectedNumberCustomRulesToBeExported = 2;
+
+    cy.get(TOASTER).should(
+      'contain',
+      `Successfully exported ${expectedNumberCustomRulesToBeExported} timelines`
+    );
+
+    cy.wait('@export').then(({ response }) => {
+      cy.wrap(response?.statusCode).should('eql', 200);
+      cy.wrap(response?.body).should('not.be.empty');
```

Comment:
As we are doing on the `Exports a custom timeline`, we should make sure that the exported timelines are the expected ones instead of making sure that the body is not empty.